automatic/git/git.nuspec

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>git</id>
+    <title>Git</title>
+    <version>2.11.0</version>
+    <authors>The Git Development Community</authors>
+    <owners>Chocolatey</owners>
+    <summary>Git or Windows offers a native set of tools that bring the full feature set of the Git SCM to Windows</summary>
```

Comment:
`Git or Windows` makes no sense: Software is called `Git for Windows`. Also remove trailing space introduced at the beginning